fungible_proxy/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "fungible_proxy"
+version = "0.0.1"
+authors = ["Request Network Foundation"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+near-sdk = "3.1.0"
+serde = "1.0.118"
+hex = "0.4"
+
+[dev-dependencies]
+near-sdk-sim = "3.2.0"
```

Comment:
near-sdk-sim 3.2.0 is deprecated in favor of [workspaces-rs](https://docs.near.org/develop/testing/workspaces-migration). Created Issue #13 to track.